src/chi/components/sidebar/sidenav.scss

```diff
@@ -376,6 +398,134 @@ $sidenav-lg-width: 12.5rem;
       }
     }
   }
+
+  &.-global-nav {
+    &.-inverse {
+      .m-sidenav__content {
+        background-color: set-color(grey, 100);
+        border-right: 0;
+
+        nav {
+          ul {
+            &.m-sidenav__list {
+              li {
+                border-bottom: 0;
+
+                &.-active {
+                  a {
+                    background-color: rgba(set-color(mint, 40), 0.08);
+
+                    &.unselected {
```

Comment:
@janibolkvadze we usually use - (dash) for this kind of classes (ie: -active). Please add it.